salt/modules/freezer.py

```diff
@@ -190,6 +190,9 @@ def restore(name=None, **kwargs):
     name
         Name of the frozen state. Optional.
 
+    clean
+        In True remove the frozen information YAML from the cache
+
```

Comment:
Please add a versionadded comment here for the new parameter. I think it's something like this:
```
.. version-added:: Sodium
```

Reply:
I have the backport to Neon in https://github.com/saltstack/salt/pull/54023, do I need to put Neon just in case?

Reply:
I also added the release note in neon